ci_scripts/travis/install.sh

```diff
@@ -24,7 +24,7 @@ popd
 
 # Configure the conda environment and put it in the path using the
 # provided versions
-conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
+conda create -n testenv --yes python=$PYTHON_VERSION pip pytest pytest-cov \
```

Comment:
It might still be worth keeping nose in the dependencies (in addition to pytest), as e.g. older versions of `numpy.testing` will use it, if I remember correctly.